bom/openhab-addons/pom.xml

```diff
@@ -471,6 +471,11 @@
       <artifactId>org.openhab.binding.freebox</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.freeathomesystem</artifactId>
```

Comment:
Please double check this is sorted alphabetically.

Reply:
Fix is done, resorted alphabetically